### PR TITLE
Adding recipe for dask_traj

### PR DESCRIPTION
**Pull request by sroet**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by sroet** [comment]
I agree to be listed in the maintainer section

**Comment by sroet** [comment]
@conda-forge/help-python this one is all ready

File: recipes/dask_traj/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  license_family: LGPL
+  # It is strongly encouraged to include a license file in the package,
+  # (even if the license doesn't require it) using the license_file entry.
+  # See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#license-file
+  license_file: LICENSE
+  summary: 'Dask based implementation of MDTraj'
+
+  # The remaining entries in this section are optional, but recommended.
+  description: |
+    Dask based implementation of MDTraj; usefull when wanting to scale out of machine. It also leads to
```
Comment (ocaisa) [inline recipes/dask_traj/meta.yaml:60]: Typo: useful

File: recipes/dask_traj/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    beter performcance in general. For a more detailed description, see package's documentation.
```
Comment (ocaisa) [inline recipes/dask_traj/meta.yaml:61]: Typo: better performance

**Review by ocaisa — COMMENTED** [review]
Some typos

File: recipes/dask_traj/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/dask_traj/meta.yaml:1]: Please delete comments here and throughout

File: recipes/dask_traj/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/dask_traj/meta.yaml:24]: delete the empty `build` section

File: recipes/dask_traj/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+    - numpy
+    - scipy
```
Comment (synapticarbors) [inline recipes/dask_traj/meta.yaml:32]: It looks like `scipy`, `distributed` and `pandas` are not actually used in the package . Note that you might want to update the upstream repo's `setup.py` to explicitly include `numpy` as a requirement

**Comment by sroet** [comment]
@synapticarbors  Thanks for the review! Should have fixed all your comments, please have another look

File: recipes/dask_traj/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
```
Comment (scopatz) [inline recipes/dask_traj/meta.yaml:22]: You probably just need pip here, not setuptools

Comment (sroet) [reply recipes/dask_traj/meta.yaml]: Good catch, thank you

**Review by dopplershift — APPROVED** [review]

**Pull request merged**